### ISSUE TITLE
Import/export session keys is not working

### ISSUE THREAD

**Issue by hiiamboris** [Nheko-Reborn/nheko#1358]
### Describe the bug

Exported and tried to import right away:
![](https://i.gyazo.com/511392244b1cbc0615f3a06d00faff09.gif)

### To Reproduce

see the gif

### Expected behavior

Import should work. Exported file seems okay, contains a base64 string.

### Version

0.11.1-02b900f8

### Operating system

Windows

### Installation method

Windows download

### Did you use profiles?

- [ ] Profiles used?

**Comment by deepbluev7** [comment]
It works fine here, so this is either a windows issue or you just had no keys to export. Did you have any encrypted messages you could decrypt?

**Comment by hiiamboris** [comment]
Encrypted messages - unlikely. I just created a new account and joined public gitter rooms.
How can I help you debug it?

**Comment by hiiamboris** [comment]
After reset #1357 all keys are marked "cached" but import is still broken.

**Comment by deepbluev7** [comment]
If you have nothing to export, then there  is nothing to import and I think that is why you get the error. The cached keys are not the session keys you are exporting with that button

**Comment by deepbluev7** [comment]
Yeah, that is probably just the metadata with an empty list encrypted by your password.

**Comment by hiiamboris** [comment]
So the real issue is just the error message?

**Comment by deepbluev7** [comment]
Probably
